app/component-library/components/Form/TextFieldSearch/TextFieldSearch.constants.ts

```diff
@@ -22,4 +23,9 @@ export const SAMPLE_TEXTFIELDSEARCH_PROPS: TextFieldSearchProps = {
   onPressClearButton: () => {
     console.log('clicked');
   },
+  size: DEFAULT_TEXTFIELD_SIZE,
+  isError: false,
+  isDisabled: false,
+  isReadonly: false,
+  placeholder: 'Sample Placeholder',
```

Comment:
nit: use sentence case
```suggestion
  placeholder: 'Sample placeholder',
```